con-base/src/main/java/ba/codecentric/base/service/ImageServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package ba.codecentric.base.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+
+@Service
+public class ImageServiceImpl implements ImageService {
+
+    @Value("${image.directory.path}")
+    private String directoryPath;
+
+    @Override
+    public String saveImage(MultipartFile image) {
```

Comment:
You are returning URL of image if upload is successful, and exception message if it fails. Hard to handle what's correct and what's wrong. Also, think about uniqueness of image name. What if image already exists having name of the image you're currently uploading?